src/main/start-main-application/lens-window/application-window/send-to-channel-in-electron-browser-window.injectable.ts

```diff
@@ -13,16 +14,16 @@ const sendToChannelInElectronBrowserWindowInjectable = getInjectable({
     () =>
       (
         browserWindow: BrowserWindow,
-        { channel, frameInfo, data = [] }: SendToViewArgs,
+        { channel, frameInfo, data }: SendToViewArgs,
       ) => {
         if (frameInfo) {
           browserWindow.webContents.sendToFrame(
             [frameInfo.processId, frameInfo.frameId],
             channel,
-            ...data,
+            toJS(data),
```

Comment:
This doesn't belong here. MobX -> toJS conversion should be done in usage.

Reply:
Why, I would have thought that doing it here helps with it being more resilient and reusable.